modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * hosted-control-planes/index.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-concepts-personas_{context}"]
+= Glossary of common concepts and personas for hosted control planes
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+[id="hosted-control-planes-concepts_{context}"]
+== Concepts
+
+hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator.
+
+hosted cluster infrastructure:: Network, compute, and storage resources that exist in the tenant or end-user cloud account.
+
+hosting cluster:: See _management cluster_.
+
+hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+management cluster:: An {product-title} cluster where the HyperShift Operator is deployed and where the control planes for tenant clusters are hosted. In some contexts, the management cluster is referred to as the _hosting cluster_.
```

Comment:
Not critical, but if these aren't totally interchangeable, just wondering if it would be worthwhile pointing out which contexts it's referred to as "hosting cluster" (and specify that in the "hosting cluster" definition, instead of just pointing here.

Reply:
They are interchangeable. I'll update the management cluster entry to make that clearer.